src/test/mockJwt.ts

```diff
@@ -8,26 +8,44 @@ const getMockJwks = (): JWKSMock =>
 
 const mockJwks = getMockJwks();
 
-const getMockJwt = (sub?: string): { Authorization: string } => {
+const getMockJwt = (
+	settings: {
```

Comment:
This isn't really a problem, but why not multiple args to `getMockJwt` instead of a `settings` map?

Reply:
@bickelj main reason both args are optional -- so multiple args would mean you have to pass unknown explicitly if the only setting you wanted was the second.  Secondary reason is that for mock generators that return a complex object I like the pattern of "pass in override values matching the ultimately mocked object" -- this isn't QUITE that just yet, but I could se it getting there.